how-to-guides/celestia-node-store-structure.md

```diff
@@ -0,0 +1,28 @@
+---
+description: This section contains information on the celestia-node datastore and its contents.
```

Comment:
does the navigation flow from previous actual page (not external link) and to the next page work as expected when you run this locally?